src/exercises/PrioritizeBookmarksToStudy.js

```diff
@@ -0,0 +1,36 @@
+import { MAX_BOOKMARKS_PER_ARTILE } from "./ExerciseConstants.js";
```

Comment:
I hadn't noticed that was the reason why some files were lowercased, but now it makes sense! 

We can do, but it would be good to add a line about it in the README then. I can do these changes.